manpages/fcpgenkey.1

```diff
@@ -11,9 +11,28 @@ fcpgenkey \- generate a single Freenet SSK@/USK@ keypair
 fcpgenkey is a simple command\-line FCP client program for generating
 new Freenet SSK@ or USK@ public/private keypairs.
 
-The keypair generated is written to standard output stream, with
-the public key, then a newline, then the private key.
+The keypair generated is written to standard output stream as SSK@
+keys, with the public key, then a newline, then the private key.
 
+.SH "Converting an SSK@ key to an USK@ one"
```

Comment:
I disagree on this one. From the POV of a user of fcpgenkey, fcpgenkey provides him with a SSK@ key but he really want a USK@ key so he needs to understand they are equivalent for his purpose.